src/main/java/io/confluent/connect/hdfs/HdfsSinkConnectorConfig.java

```diff
@@ -92,7 +96,7 @@
   public static final String TOPIC_REGEX_CAPTURE_GROUP_DOC = "A regex that specifies what groups "
       + "to capture in the topic, so when specifying `${1}` in `topics.dir`, `${1}` will refer to "
       + "the first captured group. Example config value of `[a-zA-Z]*` will capture all characters "
-      + "delimited by . _  and -, so for `topic.dir = ${1}/${2}` and `topic = topic.name.suffix` "
+      + "between [-._0-9], so for `topic.dir = ${1}/${2}` and `topic = topic.name_suffix` "
       + "the corresponding `topic.dir` will be `topic/name/`. By default, this functionality is "
```

Comment:
Since the parameter is called `topic`, maybe it would be less confusing to not have the string "topic" appear again in the example? Something like "example_name.suffix" -> "example/name" ?